src/main/webapp/src/app/find-gardens-page/find-gardens.component.ts

```diff
@@ -0,0 +1,25 @@
+import {Component, OnInit} from '@angular/core';
+import {OwlOptions} from 'ngx-owl-carousel-o';
+@Component({
+  selector: 'FindGardens',
+  templateUrl: './find-gardens.component.html',
+  styleUrls: ['./find-gardens.component.css'],
+})
+export class FindGardensComponent implements OnInit {
+  /* Option to display a certian amount of slides based off of browser window size along with
```

Comment:
grammar nit: _number_ of slides (not amount)

Reply:
Done!